app/views/super_admin/__init__.py

```diff
@@ -35,5 +35,5 @@ def check_accessible(panel_name):
     if not AuthManager.is_accessible():
         return redirect(url_for('admin.login_view', next=request.url))
     else:
-        if not current_user.can_access_panel(panel_name) or not current_user.is_staff:
+        if not current_user.can_access_panel(panel_name) and not current_user.is_staff:
```

Comment:
`current_user.is_staff` determines whether the user is an admin(with access to all panels) or not and `current_user.can_access_panel(panel_name)` checks if the user has access to that specific panel or not. Thus if a sales admin wants to access only the sales tab he will not be able to do so as `current_user.is_staff` will be false for him/her.